pages/common/pbzip2.md

```diff
@@ -0,0 +1,21 @@
+# pbzip2
+
+> A parallel implementation of the bzip2 file compressor.
+> See also: `bzip2`.
```

Comment:
```suggestion
> See also: `bzip2`, `tar`.
```

Actually, as `tar` is commonly used with it, I think it's good to mention it as well.

Reply:
✅